Exercise-HTML-CSS-1/css/styles.css

```diff
@@ -0,0 +1,62 @@
+h1{
```

Comment:
You should checkout this built-in classes to avoid writing again this kind of rules
https://getbootstrap.com/docs/4.1/utilities/text/#text-alignment